pkg/cmd/pipeline/start.go

```diff
@@ -158,25 +158,27 @@ func (opt *startOptions) getInput(pname string) error {
 		return err
 	}
 
-	if len(opt.Resources) == 0 {
-		pres, err := getPipelineResources(cs.Tekton, opt.cliparams.Namespace())
-		if err != nil {
-			fmt.Fprintf(opt.stream.Err, "failed to list pipelineresources from %s namespace \n", opt.cliparams.Namespace())
-			return err
-		}
+	if opt.Last != true {
```

Comment:
`!= true` => ` == false`

Reply:
I think this creates a bit too much nesting. can this be simplified?

Reply:
Improved a bit